easybuild/easyconfigs/t/Trilinos/Trilinos-12.12.1-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,58 @@
+name = 'Trilinos'
+version = '12.12.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://trilinos.org'
+description = """The Trilinos Project is an effort to develop algorithms and enabling technologies
+ within an object-oriented software framework for the solution of large-scale, complex multi-physics
+ engineering and scientific problems. A unique design feature of Trilinos is its focus on packages."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'usempi': True, 'pic': True, 'strict': True}
+
+source_urls = ['http://trilinos.csbsju.edu/download/files/']
```

Comment:
This URL is (maybe temporarily) unavailable. How about using the release archives in their GitHub repo?

Reply:
I don't think the tarball available via https://github.com/trilinos/Trilinos/releases is complete. I ran a `diff -ru`, and got back a whole bunch of `Only in ...: xyz` messages.

Let's see if we get any replies here: https://github.com/trilinos/Trilinos/issues/3447

Reply:
Found an alternate (probably the correct) location via our dear friend Google: https://trilinos.org/oldsite/download/files/, so I'll add that as a fallback URL.